docs/src/content/docs/basics/terminology.mdx

```diff
@@ -24,20 +28,12 @@ An assertion made about a *subject* in a *context*.
 
 ## Connection
 
-A data exchange relationship between (a) an *identity agent* and (b) an *app* or *agent*. A *connection*'s state is stored by one or more *contexts*. 
+A data exchange relationship between (a) an *identity agent* and (b) an *app*. A *connection*'s state is stored by one or more *contexts*.
```

Comment:
Looks like the term "App" includes the term "Identity agent" if I get it right, suggest removing this duplication

Reply:
Ah, good point! Thanks. I agree with this change.